src/summary/history/notes-card-style.css

```diff
@@ -30,7 +65,19 @@
   width: 70%;
 }
 
-table.table .tableDate {
+:global(.omrs-breakpoint-gt-phone) .card {
```

Comment:
Rather than setting fixed widths on bigger screens, I usually prefer to use flex. Flex offers the ability to have responsive behavior across different screen sizes.